docs/logging-and-managing-experiment-results/downloading-experiment-data.rst

```diff
@@ -0,0 +1,307 @@
+* For downloading continue reading this page.
+
+Download experiment dashboard as DataFrame
+""""""""""""""""""""""""""""""""""""""""""
+Let's download the filtered experiments dashboard view as a Pandas DataFrame, using :meth:`~neptune.projects.Project.get_leaderboard`.
+
+.. code-block::
+
+    import neptune
+
+    # Get project
+    project = neptune.init('my_workspace/my_project')
+
+    # Get dashboard with experiments contributed by 'sophia'
+    sophia_df = project.get_leaderboard(owner='sophia')
+
+    # Get another dashboard with experiments tagged 'cycleLR'
+    cycleLR_df = project.get_leaderboard(tag='cycleLR')
+
+First, you need to get correct project, then you simply run :meth:`~neptune.projects.Project.get_leaderboard` with appropriate parameters. ``sophia_df`` and ``cycleLR_df`` are pandas DataFrames where each row is an experiment and columns represent all system properties, metrics and text logs, parameters and properties in these experiments. For metrics, the latest value is returned.
+
+Note that prefixes are added to metrics, parameters and properties:
+
+* ``channel_`` for metrics and text logs, for example: ``channel_epoch/accuracy``
+* ``parameter_`` for example: ``parameter_optimizer``
+* ``property_`` for example: ``property_test_images_version``
+
+Example dataframe will look like this:
+
+.. image:: ../_static/images/logging-and-managing-experiment-results/downloading-experiment-data/data-head-2.png
+    :target: ../_static/images/logging-and-managing-experiment-results/downloading-experiment-data/data-head-2.png
+    :alt: Example downloaded dashboard data
+
+.. note::
+
+    To download only experiments that you want, you can filter them by ``id``, ``state``, ``owner``, ``tag`` and ``min_running_time``. Check :meth:`~neptune.projects.Project.get_leaderboard` documentation for details.
+
+Downloading from the experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+On this level you can use all methods that get/download data from the :class:`neptune.experiments.Experiment` object. Three types of data are especially useful: metrics, artifacts and source code.
+
+First step in all cases is to get experiment object.
+
+.. code-block:: python3
+
+    import neptune
+
+    # Get project
+    project = neptune.init('my_workspace/my_project')
+
+    # Get experiment object for appropriate experiment, here 'SHOW-2066'
+    my_exp = project.get_experiments(id='SHOW-2066')[0]
+
+Have a look at :ref:`this section <update-existing-experiment-basics-simple-example>` about updating experiments to learn more about it.
+
+Here, ``my_exp`` is :class:`neptune.experiments.Experiment` object that will be used in the following section about downloading metrics, artifacts and source code.
+
+Metrics
+"""""""
+You can download metrics data as pandas DataFrame.
+
+.. code-block:: python3
+
+    # 'my_exp' is experiment object
+    data = my_exp.get_numeric_channels_values('epoch/accuracy', 'epoch/loss')
```

Comment:
please show how this data looks (screen of data.head() )

Reply:
done